### ISSUE TITLE
Filter not working when reloading the page

### ISSUE THREAD

**Issue by cen1** [i18next/ng-i18next#100]
I am using the filter in my controller to translate some select options:
```
controller: /*@ngInject*/ ($scope, $filter) => {
$scope.optionDay = [
                { id: "*", name: $filter('i18next')('states.somestate.dayEvery') },
                { id: "1", name: $filter('i18next')('states.somestate.dayMon') },
                { id: "2", name: $filter('i18next')('states.somestate.dayTue') }
];
...
```

If I access the page via state change (menu click) it works but if I do a page reload with F5 after that the filter no longer works (everything is blank). What could be the problem?

**Comment by nsand** [comment]
I had an issue like this typically caused by the controller being instantiated before i18next was initialized.

The way that I was able to work around this was by modifying the routes (we're using ngRoute) in the project so that it had a promise to resolve before instantiating the controller. The promise is resolved by the fact of `i18n.isInitialized()` being true, or when the `i18nextLanguageChange` was broadcasted.

Check out the `resolve` property of the `route` parameter for [$routeProvider.when()](https://docs.angularjs.org/api/ngRoute/provider/$routeProvider#when)

**Comment by anwalkers** [comment]
I believe this issue is resolved in 1.0.0. i18next is initialized and locales are loaded prior to angular bootstrapping.
